cudamat/cudamat.py

```diff
@@ -142,6 +143,10 @@ def generate_exception(err_code):
         return CUDAMatException("Matrix is not in device memory.")
     elif err_code == -9:
         return CUDAMatException("Operation not supported.")
+    elif err_code == -10:
+        return CUDAMatException("Internal error.")
```

Comment:
Maybe even: `"Cannot load rnd_multipliers_32bit.txt."`. We're not going to use that error code for anything else anyway. (Regarding magic numbers: Yes, this could definitely be done better -- but we should probably leave it at that, we're just maintaining the code so it can live on, not trying to reinvent it.)